ruby_review_week_four.rb

```diff
@@ -1,37 +1,129 @@
 # You may NOT use Google. These questions are just to test where you are at. Get through as many of these as you can. If you don’t get through all of them it’s ok - they are designed to take longer than the available time. If you can’t remember a bit of code, write the pseudo-code that represents what you would write your Ruby code.
 
 # 1. What is a variable?
+A variable is a value that can change depending on the conditions passed through it by the rest of the code
```

Comment:
this is on the right track, something like this would be good

> a variable is a name that stores a value, as this value is stored we can reuse it in our program